README.md

```diff
@@ -104,6 +104,17 @@ To learn more abut what `eksctl` can do check [eksctl.io](https://eksctl.io). A
 
 Code contributions are very welcome. If you are interested in helping make `eksctl` great then see our [contributing guide](CONTRIBUTING.md).
 
+## Releases
+
+`eksctl` will be releasing on a 2 week cadence (minor releases) with extra patch releases in between as needed.
```

Comment:
I think we can avoid having to change it after 0.2.0 is out... How about something like this:
```
As of eksctl 0.2.0, minor releases should be expected every two weeks and patch release will be made available as soon as bugs get fixed. One or more release candidate(s) will be made available prior to each minor release.
```

Reply:
I think one paragraph should be sufficient here.

Reply:
Agreed.